packages/@aws-cdk-containers/ecs-service-extensions/lib/extensions/queue/lambda/lib/queue.py

```diff
@@ -0,0 +1,71 @@
+from math import ceil
+from time import time
+import json
+
+class QueueHandler:
+  def __init__(self, ecs_client, sqs_client, environ):
+    self.ecs = ecs_client
+    self.sqs = sqs_client
+    self.cluster_name = environ['CLUSTER_NAME']
+    self.service_name = environ['SERVICE_NAME']
+    self.namespace = environ['NAMESPACE']
+    self.queue_names = str(environ['QUEUE_NAMES']).split(',')
+
+  def handler(self, event, context):
```

Comment:
We don't seem to be using the parameters `event` and `context`, can we do the following?
```suggestion
  def emit(self):
```